packages/calypso-build/webpack/minify.js

```diff
@@ -125,7 +124,12 @@ module.exports = ( {
 	};
 
 	return [
-		new TerserPlugin( { parallel, extractComments, terserOptions } ),
+		new TerserPlugin( {
+			parallel,
```

Comment:
turn this off to make swcMinify go fast. it'll handle parallelization internally, rather than terser spinning up workers...

Reply:
From [docs](https://webpack.js.org/plugins/terser-webpack-plugin/#parallel)

```
Use multi-process parallel running to improve the build speed. Default number of concurrent runs: os.cpus().length - 1.
```

Just to be clear, you suggest turning `parallel: false` because webpack will handle parallelisation better?

Reply:
not webpack, rust / swc itself. You're turning off terser's parallelization stuff, which uses jest-worker or something under the hood. 

try it! it was quite a bit faster in our testing on tumblr. it may depend on how many cores your build machine has ...

Reply:
The thinking here is that swc's minifier is already parallel when running on a single file.  

So when you minimize a file, swcMinify is _already_ saturating all available cores. 

If you then run multiple swcMinify's in parallel, you're oversaturating the cores and it ends up making you slower overall.

Reply:
I pushed a commit to try this, but it didn't seem to make a difference to `CALYPSO_ENV=production yarn build` locally

Reply:
I got the same experience as Noah. I would suggest shipping this with `parallel: true` to confirm the improvements in teamcity and then iterate with `parallel: false` which might have a difference in teamcity compared to our machines.